org/ystia/yorc/yorc/linux/ansible/playbooks/templates/yorc.service.j2

```diff
@@ -22,7 +22,7 @@ After=network.target
 User=yorc
 Group=yorc
 Restart=on-failure
-ExecStart={{INSTALL_DIR}}/yorc server -c {{CONFIG_DIR}}/config.yorc.yaml
+ExecStart=/bin/bash -c 'source /etc/profile.d/proxy.sh; {{INSTALL_DIR}}/yorc server -c {{CONFIG_DIR}}/config.yorc.yaml'
```

Comment:
/etc/profile.d/proxy.sh is not standard.

I think it will be better to use Environment="http_proxy={{ansible.env.http_proxy}}" if not empty to the same for https and no proxy

Reply:
oups did not see Laurent's feedback before commenting.
But I'm definitely in favor of this solution even if you have to truncate no_proxy env

Reply:
See the truncate function in jinja http://jinja.pocoo.org/docs/2.10/templates/